src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(calculateStatistic(readFrom(fromFileName)), toFileName);
```

Comment:
```suggestion
    String fileData = readFrom(fromFileName);
    String[] statistics = calculateStatistic(fileData);
    ....
```

It's really hard to read code in this form, better to split into declaration and assignment